libs/atqr/domain/src/lib/domain-spec/stories/judge/selectNewJudge.disl

```diff
@@ -0,0 +1,65 @@
+💬 SelectNewJudge! challenge: Challenge
+    NotificationService.SelectNewJudge(challenge)
+
+💬 SupervisionRequest! challenge: Challenge, contactJudge: Contact
+    NotificationService.SupervisionRequested(challenge, contactJudge)
+    
+
+
+service NotificationService
+
+    💬 SelectNewJudge! challenge: Challenge
+        sendEmail
+          template: SelectNewJudge
+          data: Challenge
+          to: challenge.player
+        when get the response   📣 supervisionRequest
+        if(pass X days send again, if pass Y days cancel)
+
+    💬 SupervisionRequested! challenge: Challenge ,contactJudge: Contact
+        sendEmail
+          template: JudgeRequested
+          data: Challenge
+          to: contactJudge
+        when get the response   📣 judgeAccepted or judgeRejected
+        if(pass X days send again, if pass Y days cancel)
+
+
+entity User
+
+  💬 addOfficiatedChallenge! challenge: Challenge
+      officiation = new Officiation(this, challenge)
+      this.officiatedChallenges.push(officiation)
+
+      📣 addJudge! officiation: Officiation
+
+
+entity Challenge
+
+    💬 changeParticipationStatus? participationStatus: ParticipationStatus , judgeId: UserId
+        challenge.invitees.for(invitee)
+            if (invitee.id == judgeId)
+                invitee.participationStatus = participationStatus
+
+        challenge.judges.for(judge)
+            if (judge.id == judgeId)
+                judge.participationStatus = participationStatus
+
+        if(participationStatus == ParticipationStatus.ACCEPTED)
+            📣 verifyChallengeCanBegin!
```

Comment:
This should either be a Challenge's method call or an event about what happened (to be handled by a saga), not a command